test/test_images/sendevents/main.go

```diff
@@ -142,9 +140,16 @@ func main() {
 		}
 		event.SetType(eventType)
 		event.SetSource(eventSource)
-		if eventExtensionName != "" {
-			event.SetExtension(eventExtensionName, eventExtensionValue)
+
+		var eventExtensions map[string]interface{}
+		if err := json.Unmarshal([]byte(eventExtensionsMapStr), &eventExtensions); err != nil {
+			log.Println("Can't unmarshall cloud event extensions to map[string]interface{}")
```

Comment:
you can do log.Fatalf as done below, and pass the error as well. And that will end up panicking. You can remove the os.Exit(1)